modeltranslation/admin.py

```diff
@@ -43,12 +51,12 @@ def _get_declared_fieldsets(self, request, obj=None):
             return [(None, {"fields": self.replace_orig_field(self.get_fields(request, obj))})]
         return None
 
-    def formfield_for_dbfield(self, db_field, request, **kwargs):
+    def formfield_for_dbfield(self, db_field: DbField, request: HttpRequest, **kwargs: Any) -> forms.Field:  # type: ignore[return-value]
         field = super().formfield_for_dbfield(db_field, request, **kwargs)
-        self.patch_translation_field(db_field, field, request, **kwargs)
-        return field
+        self.patch_translation_field(db_field, field, request, **kwargs)  # type: ignore[arg-type]
+        return field  # type: ignore[return-value]
```

Comment:
This looks suspicious, i was expecting single `ignore[return-value]` here or on the def line.

That's why we need `warn_unused_ignores = True` for.

Reply:
Nice catch, redundant ignore removed.

By the way, the next release of `django-stubs` will remove the need for the last type ignore here